docs/plugins/ingest.asciidoc

```diff
@@ -101,9 +87,7 @@ Throws error when the field is not an array.
 --------------------------------------------------
 {
   "join": {
-    "fields": {
-      "joined_array_field": "other_array_field"
-    }
+    "joined_array_field": "-"
```

Comment:
++

I'll update the syntax for the ones that apply:

* convert
* join
* rename
* set
* split